xplique/plots/metrics.py

```diff
@@ -0,0 +1,158 @@
+"""
+Plots for metric
+"""
+
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+from ..types import Optional, Dict, Any
+
+
+def barplot(
+        scores: Dict[str, Dict[str, float]],
+        sort_metric: str = None,
+        ascending: bool = False,
+        title: str = "",
+        filepath: str = None,
+        methods_colors: Optional[Dict[str, Any]] = None
+):
+    """
+    Make a bar-plot regrouping the score of each method for different metrics
+    Used to compare the different metric scores between methods
+
+    Parameters
+    ----------
+    scores
+        Dictionary of metrics scores
+        The keys are the metric names and the values also dictionaries where
+        The keys are the method names and the values the corresponding method-metric score.
+    sort_metric
+        Name of the metric used to sort the methods (a key of the directory)
+    ascending
+        Specify the sorting direction (need a sort_metric specified)
+    title
+        Title of the graphic
+    filepath
+        Path the file will be saved at.
+        if None, the function will call plt.show()
+    methods_colors
+        Dictionary of colors for each method,
+        the values should be supported by matplotlib as a color (named colors, RGB ...)
+
+    Returns
+    -------
+
+    """
+    # pylint: disable=invalid-name
+
+    metrics = list(scores.keys())
+    # sort values
+    if sort_metric is not None:
+        methods = [
+            k for k, _ in
+            sorted(scores[sort_metric].items(), key=lambda item: item[1], reverse=not ascending)
+        ]
+    else:
+        # get methods from a metric (order do not import)
+        methods = list(scores[list(scores.keys())[0]])
+
+    # set x abscesses for each metric
+    metrics_x = {metrics[i]: i for i in range(len(metrics))}
+
+    # set x delta for each method
+    width = (1-0.2) / len(methods)
+    # -0.4 to get 0.2 between each metric, 0.5 width to center the bars
+    methods_d = {methods[i]: -0.4+(i+0.5)*width for i in range(len(methods))}
```

Comment:
I guess it is for respecting the max chars per line but there is missing spaces between operation, maybe you can go to a new line ? Otherwise, it will be rewritten with the hooker I think and won't pass the pylint CI :/